src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -52,44 +51,36 @@ internal class LanguageWorkerChannel : ILanguageWorkerChannel
         private List<IDisposable> _eventSubscriptions = new List<IDisposable>();
         private IDisposable _startSubscription;
         private IDisposable _startLatencyMetric;
-
-        private JsonSerializerSettings _verboseSerializerSettings = new JsonSerializerSettings()
-        {
-            Formatting = Formatting.Indented,
-            NullValueHandling = NullValueHandling.Ignore,
-            Converters = new List<JsonConverter>()
-                    {
-                        new StringEnumConverter()
-                    }
-        };
+        private Uri _serverUri;
 
         internal LanguageWorkerChannel()
         {
             // To help with unit tests
         }
 
-        public LanguageWorkerChannel(
-            ScriptJobHostOptions scriptConfig,
-            IScriptEventManager eventManager,
-            IWorkerProcessFactory processFactory,
-            IProcessRegistry processRegistry,
-            IObservable<FunctionRegistrationContext> functionRegistrations,
-            WorkerConfig workerConfig,
-            Uri serverUri,
-            ILoggerFactory loggerFactory,
-            IMetricsLogger metricsLogger,
-            int attemptCount)
+        internal LanguageWorkerChannel(
+           string workerId,
+           string rootScriptPath,
+           IScriptEventManager eventManager,
+           IObservable<FunctionRegistrationContext> functionRegistrations,
+           IWorkerProcessFactory processFactory,
+           IProcessRegistry processRegistry,
+           WorkerConfig workerConfig,
+           Uri serverUri,
+           ILoggerFactory loggerFactory,
+           IMetricsLogger metricsLogger,
+           bool isWebHostChannel,
+           int attemptCount)
         {
-            _workerId = Guid.NewGuid().ToString();
-
-            _scriptConfig = scriptConfig;
+            _workerId = workerId;
+            _functionRegistrations = functionRegistrations;
+            _rootScriptPath = rootScriptPath;
             _eventManager = eventManager;
             _processFactory = processFactory;
             _processRegistry = processRegistry;
-            _functionRegistrations = functionRegistrations;
             _workerConfig = workerConfig;
             _serverUri = serverUri;
-
+            _isStandbyChannel = isWebHostChannel;
```

Comment:
Naming is off/confusing on this. It's called standbyChannel, webHostChannel and placeholderChannel in various places in this file.

Reply:
I was debating which name makes more sense. renamed to webhostChannel